kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -217,6 +227,36 @@
                         ++orderInGroup,
                         Width.LONG,
                         "Line separator ByteArrayFormat");
+
+      CONFIG_DEF.define(S3_PROXY_URL_CONFIG,
+                        Type.STRING,
+                        S3_PROXY_URL_DEFAULT,
+                        Importance.LOW,
+                        "S3 Proxy settings encoded in URL syntax",
```

Comment:
Might be nice to have a bit more description on these 3 new configuration properties to help explain when/why these would be used, and how they differ from regular S3 credentials. Importance is low, so they will at least be separated from the existing credentials.

Reply:
Added more details.

Also changed username to be String (can't reply in line to the hidden comment below).